src/Initializer/TimeStepping/LtsLayout.cpp

```diff
@@ -599,9 +599,8 @@ void seissol::initializer::time_stepping::LtsLayout::normalizeClustering() {
   unsigned int l_maximumDifference      = 0;
   unsigned int l_dynamicRupture         = 0;
   unsigned int l_singleBuffer           = 0;
-  unsigned int l_totalMaximumDifference = 0;
-  unsigned int l_totalDynamicRupture    = 0;
-  unsigned int l_totalSingleBuffer      = 0;
+  // unsigned int l_totalMaximumDifference = 0;
+  // unsigned int l_totalDynamicRupture    = 0;
```

Comment:
Unwanted comments? Or was this intended to work on later?

Reply:
There was a commented message a few lines below which these variables were still for.

But—I've removed them now still: the whole file is best completely rewritten in the end, so those comments can go now as well.